docs/misc/DrawbacksOfNexusApproach.md

```diff
@@ -11,21 +11,21 @@ This takes considerable CPU and Disk resources.
 
 This typically results in an increase in disk usage of 60-100% compared to regular game size.
 
-!!! tip "For a 100GB game, an additional 60-100GB of space will be used once 'managed' with the Nexus Mods App."
+!!! tip "For a 100 GB game, an additional 60-100 GB of space will be used once 'managed' with the Nexus Mods App."
```

Comment:
```suggestion
!!! tip "For a 10 GB game, an additional 6-10 GB of space will be used once 'managed' with the Nexus Mods App."
```
The games we are actually targeting are not that big. We will likely never backup 100 GB of game files.